stations.csv

```diff
@@ -70496,3 +70496,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 74637;Bari Loseto;bari-loseto;;;41.04398202300499;16.8472726278579;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311251;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74637;
 74638;Bari Santa Rita;bari-santa-rita;;;41.07330246992781;16.851540354841326;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8311250;t;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74638;
 74639;Lascari;lascari;;;38.011695;13.928291;19906;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;S12070;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74639;
+74640;Torino Grosseto;torino-grosseto;;;45.10546665758956;7.666751284086174;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8300097;f;;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv74640;
```

Comment:
Shouldn't it be `t` for the trenitalia enabled column? (the one right after the carrier identifier)

Reply:
Also may make sense to make it suggestable

Reply:
Thanks, I have changed it now